device-orientation.html

```diff
@@ -32,15 +32,26 @@
         orientation: {
           type: Object,
           notify: true
+        },
+
+        /**
+         * Indicates whether Device Orientation API is supported or not.
+         */
+        supported: {
+          type: Boolean,
+          value: false,
```

Comment:
Please, remove the default value.
If the browser supports the Device Orientation API, we should never show `false`.